docs/internals/contributing/triaging-tickets.txt

```diff
@@ -2,16 +2,16 @@
 Triaging tickets
 ================
 
-Django uses Trac_ for managing the work on the code base. Trac is a
+Django uses Trac_ for managing the work on the codebase. Trac is a
 community-tended garden of the bugs people have found and the features people
 would like to see added. As in any garden, sometimes there are weeds to be
 pulled and sometimes there are flowers and vegetables that need picking. We need
-your help to sort out one from the other, and in the end we all benefit
+your help to sort out one from the other, and in the end, we all benefit
 together.
 
-Like all gardens, we can aspire to perfection but in reality there's no such
-thing. Even in the most pristine garden there are still snails and insects.
-In a community garden there are also helpful people who -- with the best of
+Like all gardens, we can aspire to perfection but in reality, there's no such
+thing. Even in the most pristine garden, there are still snails and insects.
+In a community garden, there are also helping people who -- with the best of
```

Comment:
Why this edit?

Reply:
Either it should be "helpful person" or helping people!

Reply:
Why can't folks be helpful? :D

Reply:
"helping people" resonates more I guess. :)
but I would revert if you wouldn't agree!

Reply:
I guess I don't see a strong reason for the change. If anything it makes this garden analogy a bit worse, because the whole point is that they _aren't_ helping the garden, despite having a helpful disposition. It's about being patient with folks who actually aren't helping sometimes.

Reply:
Also, we try to avoid trivial wording changes unless it's a big problem as it creates extra work for documentation translators.